Configuration/Services.yaml

```diff
@@ -8,4 +8,5 @@ services:
     resource: '../Classes/*'
 
   B13\Umami\Controller\StatisticsController:
+    tags: ['backend.controller']
     public: true
```

Comment:
We dont need public:true if we use the tag.

Reply:
done